### ISSUE TITLE
Feature request: ability to set Website documents sub-directory in server template

### ISSUE THREAD

**Issue by pixel-paul** [virtualmin/virtualmin-nginx#55]
In Apache, it is possible to set the 'Website documents sub-directory' in a server template:

![image](https://github.com/virtualmin/virtualmin-nginx/assets/389303/fbb7d192-6543-4762-aebe-04499ac2905e)

Could this be replicated for Nginx?

**Comment by jcameron** [comment]
Yes, this can be configured on the same page in the "Users' website subdirectory to create" field. That will apply to Nginx as well..
